index.js

```diff
@@ -0,0 +1,259 @@
+const navWrapper = document.querySelector(".nav-wrapper");
+const navToggle = document.querySelector(".mobile-nav-toggle");
+const primaryNavigation = document.querySelector(".primary-navigation");
+
+document.body.addEventListener("click", (e) => {
+  if (
+    !navWrapper.contains(e.target) &&
+    navToggle.attributes[2].nodeValue === "true"
+  ) {
+    closeNav();
+  }
+});
```

Comment:
Nice feature

Reply:
Thank you. I didn't really think of it as a feature. I just thought this is the kind of experience a user would expect.